toolkit/tools/pkg/imagecustomizerlib/customizeutils.go

```diff
@@ -346,6 +347,21 @@ func loadOrDisableModules(modules imagecustomizerapi.Modules, imageChroot *safec
 		if err != nil {
 			return fmt.Errorf("failed to write module load configuration: %w", err)
 		}
+
+		if module.Options != nil {
+			var options []string
+			for key, value := range module.Options {
+				options = append(options, fmt.Sprintf("%s=%s", key, value))
+			}
+
+			moduleOptionsFileName := module.Name + "-options.conf"
```

Comment:
Looks like we are creating an -options file for each module. Should we instead aggregate them all to one config?